wirecard/oxid-ee
# WDPD-25: Summarize Payment Results

### This PR

* Adds a column to the orders list to display the order state
* Adds a column to the orders list to display the transaction id

### TODO

* Set the order state when an order is made

### How to Test

* Login to the admin and activate the plugin
* Create a couple of orders in the frontend
* Use a database manager to connect to the table `oxorder` in the database
* Change the state from *pending* to something else for some of the entries
* Go to Administer Orders → Orders
* Confirm that the first column of the table displays the order state
* Use the filter in the table heading to limit the results to a certain order state

### Jira Links

* WDPD-25: https://jira.parkside.at/browse/WDPD-25
